index.html

```diff
@@ -14,250 +14,287 @@
     <title>Portfolio Project</title>
   </head>
   <body>
-      <header class="toolbar">
-        <a href="#" class="logo">Paul Onchera</a>
-        <nav class="navbar-container">
-          <a class="hamburger" href="javascript:void(0);"
-            ><i class="fa fa-bars"></i
-          ></a>
-          <ul class="nav-links">
-            <li>Portfolio</li>
-            <li>About</li>
-            <li>Contact</li>
-            <li><i class="fa fa-envelope-o" aria-hidden="true"></i></li>
-          </ul>
-        </nav>
-      </header>
-      <main id="class" class="grid">
-        <section class="section-test">
-          <div class="section1-details">
-            <img
-              src="images/ellipse18.png"
-              class="ellipse-image-one"
-              alt="ellipse-image-one"
-            />
-            <div class="section1-grid">
-              <div class="grid-item">
-                <img src="images/circle1.png" alt="Circle1 Image" class="circle1" />
-              </div>
-              <div class="grid-item">
-                <h1 class="header1">Paul Onchera</h1>
-                <h3 class="header3">Software Developer</h3>
-                <p class="paragraph1">
-                  I can help you build a product , feature or website Look through
-                  some of my work and experience! If you like what you see and have
-                  a project you need coded, don’t hestiate to contact me.
-                </p>
-              </div>
+    <header class="toolbar">
+      <a href="#" class="logo">Paul Onchera</a>
+      <nav class="navbar-container">
+        <a class="hamburger" href="javascript:void(0);"
+          ><i class="fa fa-bars"></i>
+          <i class="fa fa-times close-icon" aria-hidden="true"></i>
+        </a>
+        <ul class="nav-links">
+            <li class="list-item">
+              <a class="menuItem" href="#work-section">Portfolio</a>
+              <i class="fa fa-angle-right" aria-hidden="true"></i>
+            </li>
+            <li class="list-item">
+              <a class="menuItem" href="#class">About</a>
+              <i class="fa fa-angle-right" aria-hidden="true"></i>
+            </li>
+            <li class="list-item">
+              <a class="menuItem" href="#contact">Contact</a>
+              <i class="fa fa-angle-right" aria-hidden="true"></i>
+            </li>
+          <li><i class="fa fa-envelope-o envelope" aria-hidden="true"></i></li>
+        </ul>
```

Comment:
The font size of your nav links is slightly smaller than the font size in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A2925):

![image](https://user-images.githubusercontent.com/38088818/167585716-17fb936b-34c4-47d0-83ba-6db3d72863c3.png)

- Kindly increase the `font-size` of your `list-item`s to be `24px` as in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A2925):

![image](https://user-images.githubusercontent.com/38088818/167583909-309af997-4839-436e-84c9-0dde684341d0.png)